api/src/main/javadoc/doc-files/speclicense.html

```diff
@@ -20,8 +20,8 @@ <h1>Eclipse Foundation Specification License - v1.0</h1>
     <li>All existing copyright notices, or if one does not exist, a notice
         (hypertext is preferred, but a textual representation is permitted)
         of the form: &quot;Copyright &copy; [$date-of-document]
-        &ldquo;Eclipse Foundation, Inc. &lt;&lt;url to this license&gt;&gt;
-        &quot;
+        &ldquo;Eclipse Foundation, Inc. <a href="https://www.eclipse.org/legal/efsl.php">
+        https://www.eclipse.org/legal/efsl.php</a>&quot;
     </li>
```

Comment:
The above changes are not correct. These was some discussion as the initial advice was incorrect. See EL, JSP and WebSocket for correct examples.

Reply:
done